### ISSUE TITLE
Select a reference point cloud dataset

### ISSUE THREAD

**Issue by dkastl** [dotloom/data#8]
Important features:
- [ ] Open Data licensed
- [ ] Reasonable size (not too big)
- [ ] Asia (Japan) or EU would be nice
- [ ] Large enough to consist of multiple raw data files
- [ ] Good metadata
- [ ] ...

**Comment by dkastl** [comment]
@iosefa , this looks like an interesting dataset, because it not only contains the Lidar but also other geospatial data: https://grassbook.org/datasets/datasets-3rd-edition/

I know the author of the book, and this is definitely open data and probably well selected.

**Comment by iosefa** [comment]
@dkastl Thanks! I will take a look at this

**Comment by dkastl** [comment]
I guess, that they might even appreciate, if data is made available through peer-to-peer: https://www.grassbook.org/wp-content/uploads/ncexternal/
http://grass.osgeo.org/sampledata/north_carolina/

**Comment by dkastl** [comment]
Probably we just take the whole directory and make it available through DAT first. Then we can create other DATs with specific or processed content ... -> DARP2

**Comment by iosefa** [comment]
@dkastl, The data is lacking an srs. Here is the output from `pdal info --metadata`:

```bash
Josephs-MacBook-Pro:dotloom iosefa$ pdal info lidar_raleigh_nc_spm_height_feet.las --metadata
{
  "filename": "lidar_raleigh_nc_spm_height_feet.las",
  "metadata":
  {
    "comp_spatialreference": "",
    "compressed": false,
    "count": 2269213,
    "creation_doy": 0,
    "creation_year": 0,
    "dataformat_id": 0,
    "dataoffset": 227,
    "filesource_id": 0,
    "global_encoding": 0,
    "global_encoding_base64": "AAA=",
    "header_size": 227,
    "major_version": 1,
    "maxx": 645000,
    "maxy": 228500,
    "maxz": 558.87,
    "minor_version": 2,
    "minx": 633370.82,
    "miny": 215000.01,
    "minz": 88.5,
    "offset_x": 0,
    "offset_y": 0,
    "offset_z": 0,
    "project_id": "00000000-0000-0000-0000-000000000000",
    "scale_x": 0.010000000000000000208,
    "scale_y": 0.010000000000000000208,
    "scale_z": 0.010000000000000000208,
    "software_id": "libLAS 1.7.0",
    "spatialreference": "",
    "srs":
    {
      "compoundwkt": "",
      "horizontal": "",
      "isgeocentric": false,
      "isgeographic": false,
      "prettycompoundwkt": "",
      "prettywkt": "",
      "proj4": "",
      "units":
      {
        "horizontal": "unknown",
        "vertical": ""
      },
      "vertical": "",
      "wkt": ""
    },
    "system_id": "libLAS"
  },
  "pdal_version": "1.7.1 (git-version: b6474d)"
}
```

You can also see this at: https://www.grassbook.org/wp-content/uploads/ncexternal/lidar_raleigh_nc_spm_height_feet.txt

But... the spatial reference is defined somewhere else on the website. So maybe it will be a god example of DARP.

**Comment by iosefa** [comment]
If we use this data (and I dont see a strong enough reason to not use it), we need to preprocess it to have the correct SRS employed. Which is a good example of how we can use DARP2. So, I will first update documentation on DARP2

**Comment by iosefa** [comment]
@dkastl let me know when it is done. Can we share the dat link on this repo? If so we should add it to the readme as the source of example data.

**Comment by dkastl** [comment]
Yes, I will share the link, when it's done. But it's still downloading ... currently at 1.2GB for the `north_carolina` folder. DARP2 will be definitely useful here to document derived DAT archives.

**Comment by iosefa** [comment]
Yes. This is a great use case of DARP2

**Comment by dkastl** [comment]
The GRASS North Carolina dataset is now available on 4 persistent nodes under the following Dat archibe link:

`dat://e2e0d0b38f66b51f79441912194c9434d9748168958c28edf2c45ff4fbb6f66f
`

The node locations are Germany, Finland and Japan at the moment.

**Comment by dkastl** [comment]
If you get memory leak warnings, this can help:
```
env DISABLE_NANOTIMING=true dat sync
```
